libr/core/cmd_write.c

```diff
@@ -1788,3 +1843,63 @@ static int cmd_write(void *data, const char *input) {
 	R_FREE (ostr);
 	return 0;
 }
+
+static void cmd_write_init(RCore *core, RCmdDesc *parent) {
+	DEFINE_CMD_DESCRIPTOR (core, w);
+	DEFINE_CMD_DESCRIPTOR (core, wa);
+	DEFINE_CMD_DESCRIPTOR (core, wA);
+	DEFINE_CMD_DESCRIPTOR (core, wc);
+	DEFINE_CMD_DESCRIPTOR (core, we);
+	DEFINE_CMD_DESCRIPTOR (core, wo);
+	DEFINE_CMD_DESCRIPTOR (core, wop);
+	DEFINE_CMD_DESCRIPTOR (core, wp);
+	DEFINE_CMD_DESCRIPTOR (core, wt);
+	DEFINE_CMD_DESCRIPTOR (core, wv);
+	DEFINE_CMD_DESCRIPTOR (core, wx);
+
+	RCmdDesc *wv_cd = r_cmd_desc_argv_new (core->rcmd, parent, "wv", wv_handler);
```

Comment:
Shouldn't the parent of `wv` be `w` instead of root?

Reply:
There is no `w` node. Everything is handled by the root node, apart from `wv` (for now).
Of course we could also create a new old_input node `w`, and make `wv` a child of `w`, but at this point I did not want to touch too many things. Mainly because if I'm going to change things, I would prefer to directly invest the time in converting them to arg nodes.

Reply:
The reason I asked because in my opinion it is a valuable example for how partially converting things to new system will work. More or less the same as question I asked lower. You could put effort into fully converting `w`, but it will not be possible everything at the same time. So partial conversion process should be validated early to ensure that design can support it well.

Reply:
So you want to try having `w` as a oldinput node as child of `root`? With `wv` as child of `w`?